src/hooks/use-event-data.js

```diff
@@ -49,6 +50,23 @@ export const sampleEvents = [
     },
 ];
 
+export const removePastEvents = events => {
+    const today = createDateObject(new Date());
+    return events.filter(
+        e =>
+            createDateObject(new Date(e.node_type_attributes.event_end)) >=
+            today
+    );
+};
+export const sortEvents = events =>
+    // create new list first so original does not get mutated
+    // this improves testability and reduces side-effects
+    [...events].sort((a, b) => {
```

Comment:
I knew we had spoken briefly about this, but seems like we could get sorted events with ` https://www.mongodb.com/api/event/all/1?sort=-node_type_attributes.event_start`